packages/adminrouter/extra/src/README.md

```diff
@@ -490,16 +508,16 @@ It exposes a couple of targets:
   remove the images themselves though as the layer cache may be useful later
   on and the user may remove them themselves.
 * `make devkit` - creates the `adminrouter-devkit` container. By default other
-   targets execute this step automatically if devkit container image does not
-   exist yet.
+  targets execute this step automatically if devkit container image does not
+  exist yet.
 * `make update-devkit` - updates `adminrouter-devkit`. Should be run every time
-   the Dockerfile or its dependencies change.
+  the Dockerfile or its dependencies (e.g. requirements.txt fileds) change.
 * `make tests` - launch all the tests. Worth noting is the fact that McCabe
-   complexity of the code is also verified, and an error is raised if it's
-   equal to or above 10.
+  complexity of the code is also verified, and an error is raised if it's
+  equal to or above 10.
 * `make shell` - launch an interactive shell within the devkit container. Should
   be used when fine grained control of the tests is necessary or during debugging.
-* `make flake8` - launch flake8 which will check all the tests and test-harness
+* `make lint` - launch flake8 which will check all the tests and test-harness
```

Comment:
aaaahhhh... I misunderstood you - I thought you meant "Change flake8 to linters" in the `make flake8` part!

Fixing!